pkg/security/probe/probe_linux.go

```diff
@@ -1196,6 +1205,36 @@ func (p *Probe) FlushDiscarders() error {
 	return bumpDiscardersRevision(p.Erpc)
 }
 
+// FlushSyscalls does a best effort flush of the in-flight syscalls map
+func (p *Probe) FlushSyscalls() {
+	seclog.Debugf("Flushing in-flight syscalls")
+
+	m := p.syscallsMap
+
+	if m == nil {
+		seclog.Errorf("flushing syscalls: nil-map")
+		return
+	}
+
+	var (
+		key   uint64
+		value [352]byte
```

Comment:
is 352 the number of syscalls that exist or syscalls that we monitor?

Reply:
ah sorry should'n have undrafted this since I'm still playing with it. 352 is `sizeof syscall_cache_t` for now, but I need to clean this up (ensure this stays in sync etc, if I actually want to merge this)